README.md

```diff
@@ -16,19 +20,20 @@ Each service might have a number of libraries available from each of the followi
 
 ### Client: New Releases
 
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview** on `npm`. These libraries allow you to use and consume existing resources and interact with them. These libraries share a number of core functionalities found in the Azure Core package such as retries, logging, transport protocols, authentication protocols, etc. Learn more about these libraries by reading [the guidelines](https://azure.github.io/azure-sdk/typescript/guidelines/) that they follow.
+Given an Azure resource already exists, you would use the client libraries to consume it and interact with it.
```

Comment:
I'm not sure how this relates to the header "New Releases"? It's also said above so maybe this could just be deleted.

Reply:
I believe this was an early attempt to differentiate between the Track 1 and Track 2 libraries. Honestly, I don't see why one should care

Reply:
I agree that I would imagine track2 is not "new" anymore enough that this:

```
- [Client - New Releases](#client-new-releases)
- [Client - Previous Versions](#client-previous-versions)
```

could be this?

```
- [Client](#client)
- [Client - Legacy Versions](#client-legacy-versions)
```

Reply:
But why would a reader need to know that when reading the root readme at all?
The sub folder readme for the legacy package as well as the readme in npm will be noting if the library is legacy/deprecated. What do we gain by making that differentiation here?

Reply:
After offline discussions, we decided to go with just client vs management. Corresponding change was pushed in https://github.com/Azure/azure-sdk-for-js/pull/13255/commits/4da82a8ee5e14172d3a42989b900ba494c821591